### ISSUE TITLE
Numbers values are always zero

### ISSUE THREAD

**Issue by lkraider** [apiaryio/drafter#334]
For:
```
drafter --version
v2.1.1
```

Given the following input:
```
# Data Structures
## Crop
+ x: ​5 (number)
+ y: ​10 (number)
+ w: ​625 (number)
+ h: ​500 (number)
```

The values content for the numbers are always zero after parsing:
```
element: "parseResult"
content:
  -
    element: "category"
    meta:
      classes:
        - "api"
      title: ""
    content:
      -
        element: "category"
        meta:
          classes:
            - "dataStructures"
        content:
          -
            element: "dataStructure"
            content:
              -
                element: "object"
                meta:
                  id: "Crop"
                content:
                  -
                    element: "member"
                    content:
                      key:
                        element: "string"
                        content: "x"
                      value:
                        element: "number"
                        content: 0
                  -
                    element: "member"
                    content:
                      key:
                        element: "string"
                        content: "y"
                      value:
                        element: "number"
                        content: 0
                  -
                    element: "member"
                    content:
                      key:
                        element: "string"
                        content: "w"
                      value:
                        element: "number"
                        content: 0
                  -
                    element: "member"
                    content:
                      key:
                        element: "string"
                        content: "h"
                      value:
                        element: "number"
                        content: 0
```

**Comment by lkraider** [comment]
Inspecting the text file with an hex editor, I noticed there were these `E2 80 8B` invisible characters in front of the numbers. Removing them made the parser identify the content correctly.

Sample bad line / hex:
```
+ x: ​10 (number)
```
```
2B 20 78 3A 20 E2 80 8B 31 30 20 28 6E 75 6D 62 65 72 29 0A
```

Sample "good" line / hex:
```
+ x: 10 (number)
```
```
2B 20 78 3A 20 31 30 20 28 6E 75 6D 62 65 72 29 0A
```

**Comment by pksunkara** [comment]
If it is has any other characters other than numbers, then drafter will parse it as a zero.

**Comment by kylef** [comment]
@pksunkara I think we should start showing a warning when the user puts in non-numeric content to numbers instead of silently dropping so I'm going to re-open.

**Comment by pksunkara** [comment]
Fixed by #376
